test/dlc_tests/eks/eks_manifest_templates/kubeflow/install_kubeflow.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+set -e
+
+install_kfctl(){
+    #install kfctl cli
+    KFCTL_VERSION="v1.0.2"
+
+    if ! command -v kfctl &> /dev/null
+    then
+        echo "not installed"
+        KFCTL_URL=https://github.com/kubeflow/kfctl/releases/download/${KFCTL_VERSION}/kfctl_${KFCTL_VERSION}-0-ga476281_linux.tar.gz
+        curl --silent --location ${KFCTL_URL} -o /tmp/kfctl_${KFCTL_VERSION}_linux.tar.gz
+        tar -xvf /tmp/kfctl_${KFCTL_VERSION}_linux.tar.gz -C /tmp --strip-components=1
+        mv /tmp/kfctl /usr/local/bin
+    fi
+}
+
+setup_kubeflow(){
+    #install kubeflow in EKS cluster
+    REGION=$2
+    KUBEFLOW_URL="https://raw.githubusercontent.com/aws/deep-learning-containers/master/test/dlc_tests/eks/eks_manifest_templates/kubeflow/kfctl_aws.yaml"
+    CONFIG_FILE=kfctl_aws.yaml
+    wget -O ${CONFIG_FILE} ${KUBEFLOW_URL} 
+    sed -i -e 's/<REGION>/'"$REGION"'/' ${CONFIG_FILE}
+    kfctl apply -V -f ${CONFIG_FILE}
+}
+
+install_mpi_operator() {
+    #install mpi operator in EKS cluster
+    MPI_OPERATOR_URL=https://raw.githubusercontent.com/kubeflow/mpi-operator/v0.2.3/deploy/v1alpha2/mpi-operator.yaml
+    wget -O mpi-operator.yaml ${MPI_OPERATOR_URL}
+    kubectl create -f mpi-operator.yaml
+}
+
+install_mxnet_operator() {
+    #install mxnet operator in EKS cluster
+    git clone https://github.com/kubeflow/mxnet-operator.git
+    kubectl create -k mxnet-operator/manifests/overlays/v1beta1/
+}
+
+create_dir(){
+
+    DIRECTORY="$HOME/$1"
```

Comment:
can you give hints as to what the $1 and $2 args are, maybe by assigning them to variables in this file iteself?

Reply:
will add.